cypress/integration/main_page_spec.js

```diff
@@ -281,5 +281,13 @@ describe('Dashboard Home Page', () => {
 
       shouldHaveDeletedToken();
     });
+
+    it('adds the default datepicker options to the URL', () => {
+      const now = Date.UTC(2009, 0, 5, 13, 35, 3);
+      cy.clock(now);
+
+      cy.visit('/');
+      cy.url().should('include', 'startDateTime=20090104133500');
+    });
   });
```

Comment:
We should also have the test where you visit('/'), "clear all" then click to granules (not "visit"), then click back to home and ensure the url is free of datetime QPs.

Reply:
Added

Reply:
sorry totally missed that in the last go around.